frontend/src/AppBuilder/QueryManager/Components/QueryManagerBody.jsx

```diff
@@ -330,7 +346,14 @@ export const QueryManagerBody = ({ darkMode, options, setOptions, activeTab }) =
       }} // 40px for preview header height
     >
       {selectedDataSource === null || !selectedQuery ? (
-        renderDataSourcesList()
+        showLocalDataDeprecationBanner ? (
+          <>
+            <NotificationBanner />
```

Comment:
Does support dark mode styles automatically? If yes, please add the dark mode variable inside the component itself or pass value from the referenced files.